src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -1270,7 +1270,12 @@ internal void CompileAndExecuteMSIL(List<AssociativeNode> finalDeltaAstList)
         {
             Dictionary<string, IList> input = new Dictionary<string, IList>();
             var assemblyPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
-            var codeGenIL = new EmitMSIL.CodeGenIL(input, Path.Combine(assemblyPath, "opCodes.txt"));
+
+            //TODO_MSIL: remove the dependency on the old VM by implementing
+            //necesary Emit functions(ex mitFunctionDefinition and EmitImportStatements and all the preloading logic)
+            msilRuntimeCore = msilRuntimeCore ?? new MSILRuntimeCore(runtimeCore);
```

Comment:
is there a jira link?

Reply:
not yet...I will add the to the PR description